packages/webview_flutter/webview_flutter_web/README.md

```diff
@@ -0,0 +1,20 @@
+# webview\_flutter\_web
+
+This is an implementation of the [`webview_flutter`](https://pub.dev/packages/webview_flutter) plugin for web.
+
+It is currently severely limited and doesn't implement most of the available functionality.
+The following functionality is currently available:
+
+- `loadUrl` (Without headers)
+- `requestUrl`
+- `loadHTMLString` (Without `baseUrl`)
+- Setting the `initialUrl` through `CreationParams`.
+
+Nothing else is currently supported.
+
+## Usage
+
+### Depend on the package
+
+This package is not an endorsed implementation of the google_maps_flutter plugin yet, so you'll need to
```

Comment:
s/google_maps_flutter/`webview_flutter`/ (with backticks around the corrected name for consistency with the first line of the README).